src/blocks/app-and-main-menu/_fhi-main-menu.block.scss

```diff
@@ -83,19 +88,39 @@
                 padding: $fhi-brand-height-xs 0 $fhi-space-7;
             }
             &__nav-link {
-                border-radius: $fhi-px * 4;
                 color: $fhi-main-menu-xs-color;
-                font-size: $fhi-font-size-3;
-                text-decoration: none;
+            }
+        }
+    }
+}
 
-                &:hover {
-                    background-color: $fhi-blue-dark-2;
-                }
+
+.fhi-main-menu {
+    .nav-link {
```

Comment:
Don't style on Bootstrap classes, use BEM

Reply:
This is just temporarily. The proper styling will be part of the Global Header PR